Dockerfile

```
@@ -53,6 +56,8 @@ RUN set -xe \
     && curl -Ls https://archive.apache.org/dist/xmlgraphics/fop/binaries/fop-2.1-bin.tar.gz | tar xz -C /usr/share \
     && ln -sf /usr/share/fop-2.1/fop /usr/local/bin/fop
 
+RUN echo "extension=ldap.so" > /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini
```

It's best to append to the previous RUN, saves you from producing an extra image layer.

```suggestion
    && ln -sf /usr/share/fop-2.1/fop /usr/local/bin/fop \
    && echo "extension=ldap.so" > /usr/local/etc/php/conf.d/docker-php-ext-ldap.ini
```

This is a good article on this topic: https://emmer.dev/blog/reducing-docker-layers/#combine-multiple-run-instructions.